server/models/Board.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require("mongoose");
+
+const boardSchema = new mongoose.Schema({
+  title: {
+    type: String,
+    required: true,
+    unique: false,
+  },
+  columns:[
+    {
+      type: String,
+    }
+  ]
```

Comment:
Wouldn't we be doing a reference store here? So in that case, we would want to do a ObjectId type with a `ref: "column"` so we are indiciating the ObjectId should be of type Column.

Reply:
Got it! Making that edit.